CHANGELOG.md

```diff
@@ -1,12 +1,17 @@
 # v0.5.0 (Unreleased)
+### BUG FIXES
+* **OAuth Client Resource:** Resolved issue where default issues on the `pingfederate_oauth_client` resource cause an invalid apply ([#](https://github.com/pingidentity/terraform-provider-pingfederate/pull/156)) 
```

Comment:
Missing the number on the link text. Also is this supposed to say "default issues"?